### PR TITLE
Add documentation for mdot-fu and add to fancy contrib

### PR DESCRIPTION
**Pull request by csziacobus**

### CODE REVIEW

**Comment by luismbo** [comment]
Since mdot-fu has been disabled for so long, I'm inclined to be conservative and only enable it by default after a month or two of daily testing. Do you agree?

**Comment by csziacobus** [comment]
Sure, that's more than reasonable. I guess we can just merge the documentation then so people will know it exists and can test.

**Comment by luismbo** [comment]
Oh, sure. Care to rewrite the patch to keep the documentation bits only?

File: doc/slime.texi [hunk 1]
```diff
@@ -3177,6 +3178,11 @@ Toggle exclusion of swank functions from the report.
 
 @end table
 
+@node SLIME Enhanced M-.
+@section SLIME Enhanced M-.
+@code{slime-mdot-fu} enables meta-point to jump to local variables bound with @code{let} and @code{let*},
```
Comment (luismbo) [inline doc/slime.texi]: I think you should `M-x fill-paragraph` this. (And I think slime.texi assumes `fill-column` = 70.)

**Comment by csziacobus** [comment]
Commit amended.

**Comment by luismbo** [comment]
Commited via 72701d5. I noticed a couple of days ago that, unlike what you (and I) expected, slime-mdot-fu works for `LET` but not `LET*`. I believe I've fixed it in 49673e9. Thanks!

**Pull request closed without merging**